File: 프로그래머스/lv1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.java
class Solution {
    public long[] solution(long x, int n) {
        long[] arr = new long[n];
        int count = 0;
        for (long i = x; count < n; i += x) {
            arr[count++] = i;
        }
        
        return arr;
    }
}

File: 프로그래머스/lv1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.py
def solution(x, n):
    if n == 0:
        return []
    if x == 0:
        return [0 for _ in range(n)]
    if x < 0:
        return [i for i in range(x, x * n - 1, x)]
    else:
        return [i for i in range(x, x * n + 1, x)]